cli/cmd/plugin/standalone-cluster/kapp/kapp.go

```diff
@@ -119,25 +131,26 @@ func (k KappClient) Install(opts KappInstallOpts) (*v1.Deployment, error) {
 		if createdObj.GetObjectKind().GroupVersionKind().Kind == deploymentKind {
 			if createdObj.Object[metadataKey].(map[string]interface{})[nameKey] != nil {
 				if createdObj.Object[metadataKey].(map[string]interface{})[nameKey].(string) == kappControllerDeploymentName {
-					runtime.DefaultUnstructuredConverter.FromUnstructured(createdObj.Object, &kappDeployment)
+					_ = runtime.DefaultUnstructuredConverter.FromUnstructured(createdObj.Object, &kappDeployment)
 				}
 			}
 		}
 	}
 	return &kappDeployment, nil
 }
 
+// Status gets the status of a package.
 func (k KappClient) Status(ns, name string) string {
-
 	pods, err := k.clientSet.CoreV1().Pods(ns).List(metav1.ListOptions{})
 	if err != nil {
 		return "failed to talk to cluster"
 	}
 
 	var kappPod *corev1.Pod
 	// super bad, should not be iterating through pods on every invocation
-	for _, pod := range pods.Items {
+	for i := range pods.Items {
```

Comment:
hmmm was the linter failing on `for _, pod`? Seems weird to have to pull the item out manually using the index instead of the slice item itself

Reply:
This is why I like this lint check. It's actual a very subtle bug, and not something you would expect based on just looking at the code. But when you get the pointer to the variable, you actually end up with the pointer to the loop variable and not the actual item. So even doing something like `pod := pod` fixes that, but looks like WTH.

https://github.com/golang/go/wiki/CommonMistakes#using-reference-to-loop-iterator-variable

Reply:
This is why I like this lint check. It's actual a very subtle bug, and not something you would expect based on just looking at the code. But when you get the pointer to the variable, you actually end up with the pointer to the loop variable and not the actual item. So even doing something like `pod := pod` fixes that, but looks like WTH.

https://github.com/golang/go/wiki/CommonMistakes#using-reference-to-loop-iterator-variable